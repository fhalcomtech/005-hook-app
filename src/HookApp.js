import { CounterApp } from "./Component/001-use-state/CounterApp";
import { CounterWithCustomHook } from "./Component/001-use-state/CounterWithCustomHook";
import { FormWithCustomHook } from "./Component/002-use-effect/FormWithCustomHook";
import { SimpleForm } from "./Component/002-use-effect/SimpleForm";
import {MultipleCustomHook} from "./Component/002-use-effect/MultipleCustomHook";

const HookApp = () => (<>
        <MultipleCustomHook/>
    </>);
export default HookApp;
