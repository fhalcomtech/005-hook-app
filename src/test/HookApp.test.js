import React from "react"
import HookApp from "../HookApp";
import {shallow} from "enzyme";

describe("Making test with HookApp Component", ()=>{
    test("Write Snapshot", ()=>{
       const wrapper = shallow(<HookApp/>);
       expect(wrapper).toMatchSnapshot();
    });
})