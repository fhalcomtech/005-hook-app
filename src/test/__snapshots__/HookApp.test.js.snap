// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Making test with HookApp Component Write Snapshot 1`] = `
<div>
  <h1>
    Hola Mundo
  </h1>
</div>
`;
